Px.Dcat/DataCollector.cs

```diff
@@ -541,7 +541,7 @@ private Organization getProducer(PXMeta meta, List<string> langs)
                 }
 
                 newOrg.Names = names;
-                newOrg.Resource = _settings.BaseUri + "organization/" + nextString();
+                newOrg.Resource = Path.Combine(_settings.BaseUri,"organization",nextString().Replace("\\", "/"));
```

Comment:
You only replace backslashes in get nextString() string. You should move it outside of the parenthesis of Path.Combine.